examples/sshd.yml

```diff
@@ -14,6 +14,10 @@ services:
     image: "linuxkit/rngd:1fa4de44c961bb5075647181891a3e7e7ba51c31"
   - name: dhcpcd
     image: "linuxkit/dhcpcd:7d2b8aaaf20c24ad7d11a5ea2ea5b4a80dc966f1"
+  - name: getty
+    image: "linuxkit/getty:148946d72d1c96df3ea91cb8ee4f9583cd3cc5c2"
+    env:
+     - INSECURE=true
   - name: sshd
```

Comment:
I prefer to leave it, if we can. If someone is logging in, let's give them the console as well.